recipes/compute.rb

```diff
@@ -62,19 +62,19 @@
       'grep -E \'SMT is off|Machine is not SMT capable\''
   end
 when 'x86_64'
-  kernel_module 'kvm_intel' do
+  kernel_module 'kvm-intel' do
     onboot true
     reload false
     options %w(nested=1)
-    check_availability true
-    only_if { node.deep_fetch('dmi', 'processor', 'Manufacturer') == 'Intel(R) Corporation' }
+    action  :load
```

Comment:
This is the default action, why set it?

Reply:
@ramereth ah yes. I thought `configure` was the default for some reason.